.github/workflows/docker-image.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Docker Image CI
+on:
+  - push
+  - pull_request
```

Comment:
please use `pull_request_target` as opposed to `pull_request` so we can support pull requests from forks. 

Reference: https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/

Reply:
Removed the proposed Github Action for now as this change does not directly impact the core point of the PR -> removing the NSP definition.